src/windows/new_project.rs

```diff
@@ -239,21 +239,17 @@ impl Window {
 
         progress.zip_total.store(zip_url.len(), Ordering::Relaxed);
 
-        let zips = futures::future::join_all(zip_url.iter().map(|url|
-            // surf::get(format!("https://api.allorigins.win/raw?url={url}"))  FIXME: phishing scam, apparently
-            surf::get(url)
-            .middleware(surf::middleware::Redirect::new(10))))
-        .await;
+        let zips = futures::future::join_all(zip_url.iter().map(|url| reqwest::get(*url))).await;
```

Comment:
Does this work with CORS?

Reply:
Probably, coi-serviceworker.js is set up to use COEP in "credentialless" mode so it should be able to download files from other websites